pro/src/screens/OfferIndividual/StocksEventCreation/__specs__/RecurrenceForm.spec.tsx

```diff
@@ -13,6 +14,11 @@ const defaultProps = {
 }
 
 describe('RecurrenceForm', () => {
+  it('should have no basic accessibility issues', async () => {
```

Comment:
peut-être plutôt `should pass axe accessibility tests` ? (ça n'exclut pas qu'on ait des problèmes d'accessibilité basiques)